var/spack/repos/builtin/packages/petaca/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Petaca(CMakePackage):
+    """A collection of modern Fortran modules.
+
+    Modules include:
+    - Parameter lists
+    - map_any_type
+    - fortran_dynamic_loader
+    - timer_tree_type
+    - yajl_fort
+    - json
+    """
+
+    homepage = "https://petaca.readthedocs.io/en/master"
+    git      = "https://github.com/nncarlson/petaca.git"
+
+    maintainers = ['pbrady']
+
+    version('develop', branch="master")
+    version('1.0', commit='f17df95193ca1a3879687a59a91a123be25e3efa', preferred=True)
```

Comment:
It looks like the project doesn't have a release scheme or any version numbering. If this is just an arbitrary version then use the date as a substitute version number (`develop-2021-03-31` might be best).

Reply:
Thanks for the review.  It is an arbitrary version so I went with the date as suggested.